compiler/ballerina-parser/src/test/resources/statements/assignment-stmt/assignment_stmt_source_12.bal

```diff
@@ -1,5 +1,5 @@
 function foo() {
     a =
-    int b =
+    b =
     int a = 6;
 }
```

Comment:
With `let expression` support this `assignment-stmt` recovery test case recovers as a let expression. So I altered the test case a little.  Is it alright?

Reply:
Lets create an issue.